packages/elements/src/assets/ino-icon/code_block.svg

```diff
@@ -1,14 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="22.521" height="20.409" viewBox="0 0 22.521 20.409">
```

Comment:
Compared by rich diff this icon is now broken.

Reply:
Can confirm, can be seen here: https://elements.inovex.de/unicorn/elements-icons-chore-auto-optimize-icons/?path=/docs/graphic-ino-icon--all-icons